### ISSUE TITLE
[Keyboard navigation - Make code - Gettings started tour]: Keyboard focus moves out of the 'Welcome' dialog while navigating using tab key.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6095]
### User Experience:
Keyboard dependent people will get impacted if keyboard focus moves out of the dialog while navigating using tab key. Due to which the end user requires an extra effort to reach the dialog which is time consuming.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/beta#.
2. Home page will open.
3. Navigate to 'Create project' button and select it. Editor page appears.
4. Getting started tour pop up appears, navigate the page and observe whether keyboard focus moves out of the 'Welcome' dialog or not.

### Actual Result:
**Issue:** Keyboard focus moves out of the 'Welcome' dialog while navigating using tab key.

**Observation:**
After 'Next' button, by pressing tab key, keyboard focus moves out of the page.

### Expected Result:
Keyboard focus should not move out of the 'Welcome' dialog while navigating using tab key.

**Example:** Aria-modal="true" should be defined.

### MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Yes (Refer attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- **OS Version:** Windows 11 24H2 (OS Build:26100.3037)
- **Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
- **Test Matrix:** Chrome + NVDA
- **URL:** [https://makecode.microbit.org/.](https://makecode.microbit.org/beta#)

### Attachment:

<img width="980" alt="Image" src="https://github.com/user-attachments/assets/d0d5bbed-14c4-451b-b908-577d519bd427" />

**Comment by vroopea** [comment]
#Accessibility; #A11yMAS; #A11ySev2; #MAS2.4.3; #AILimited; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025
